voxxed-microservices-2018/module-01.adoc

```diff
@@ -253,11 +253,20 @@ $ export NAMESPACE=voxxed
 $ oc create sa prometheus-server
 $ oc adm policy add-cluster-role-to-user cluster-reader system:serviceaccount:${NAMESPACE}:prometheus-server
 
-Finally, create the Prometheus service by running:
+Create the Prometheus service by running:
 
 [source,sh]
 $ oc apply -f https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/master/metrics/examples/prometheus/kubernetes.yaml
 
+Finally it is necessary to enable metrics collection from Kafka brokers by definining a set of data available from JMX.
+This is configured in the Kafka resource.
+
+ * Open the kafka resource using command `oc edit kafka my-cluster`.
```

Comment:
Pasting this in seems a bit brittle and easy to get wrong. How about providing the entire resource and simply overwrite the existing one?

Reply:
I was thinking about patching but I've not found yet a way how to patch from a file

Reply:
@gunnarmorling How about something like `oc get kafka my-cluster -o yaml | sed '/kafka:/ r kafka-metrics.yaml' - | oc apply -f -`